### PR TITLE
Add `encodeURIComponent` for search queries

### PR DESCRIPTION
**Pull request by filoozom**
Basically the same as #63 but rebased. Please publish a new version after this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "app-store-scraper",
-  "version": "0.11.0",
+  "version": "0.11.1",
```
Comment (facundoolano) [inline package.json]: don't bump the version, I do this when I publish to generate the proper tag

**Comment by filoozom** [comment]
And this is a breaking change actually so might need to bump the major.

**Comment by filoozom** [comment]
Because people might already be using `encodeURI` or `encodeURIComponent` on their side, which would now double-encode the search term:
```
encodeURIComponent('&')
"%26"
encodeURIComponent('%26')
"%2526"
```

**Comment by facundoolano** [comment]
gotcha.

**Comment by facundoolano** [comment]
Published. We're still under 1.0.0, so no major bump.

**Pull request merged**